packages/flutter/test/widgets/text_selection_test.dart

```diff
@@ -522,6 +522,34 @@ void main() {
     }
   }, variant: TargetPlatformVariant.all());
 
+   testWidgets('toolbar state is correct on each platform when tapping focused editable', (WidgetTester tester) async {
+    await pumpTextSelectionGestureDetectorBuilder(tester);
+    final FakeEditableTextState state = tester.state(find.byType(FakeEditableText));
+    final FakeRenderEditable renderEditable = tester.renderObject(find.byType(FakeEditable));
+    renderEditable.hasFocus = true;
+
+    final TestGesture gesture = await tester.startGesture(
+      const Offset(200.0, 200.0),
+      pointer: 0,
+    );
+    addTearDown(gesture.removePointer);
+    await gesture.up();
```

Comment:
Could this just be a `tapAt` or is there something different about doing it this way?

Reply:
Updated to use `tapAt`.